src/containers/NewHostContainer/__tests__/index_test.js

```diff
@@ -38,7 +38,17 @@ test('Mounted NewHostContainer', async () => {
   const customPuppetConfigButton = wrapper.find('button').findWhere(x => x.text() === 'hosts_form.puppet_config.default.link')
   customPuppetConfigButton.simulate('click')
 
-  await wait(0)
+  await wait()
+  wrapper.update()
+
+  const puppetEnvIdSelect = wrapper.find('Select[placeholder="hosts_form.placeholders.puppet_env_id"]')
+  puppetEnvIdSelect.instance().selectValue({ value: 'MDE6RW52aXJvbm1lbnQtMA==', label: 'env2' })
+
+  await wait()
+  wrapper.update()
+
+  const puppetclassIdsSelect = wrapper.find('Select[placeholder="hosts_form.placeholders.puppetclass_ids"]')
+  puppetclassIdsSelect.instance().selectValue({ value: 'MDE6UHVwcGV0Y2xhc3MtMg==', label: 'ppclass2' })
```

Comment:
`environmentId` and `puppetclassIds` are now selected "manually" (other than the default values)